pkg/events/parse_args_helpers.go

```diff
@@ -46,7 +46,7 @@ func parseBPFProgType(arg *trace.Argument, progType uint64) {
 	arg.Type = "string"
 	bpfProgTypeArgument, err := parsers.ParseBPFProgType(progType)
 	if err != nil {
-		arg.Value = ""
+		arg.Value = progType
```

Comment:
If the raw value is used, the value type is inconsistent. We should either change the type accordingly or (more preferrable in my opinion) format the raw value as a string.

Reply:
I agree - better to format the raw argument as a string

Reply:
Yes, that sounds good. However, as discussed earlier, raw values are used when Tracee can't identify the data.  In this code, forming the raw value isn't the best practice because that happens in the pkg/parsers module.

Reply:
This should work: `fmt.Sprint(rawVal)`